imagebuildah/stage_executor.go

```diff
@@ -1903,6 +1903,10 @@ func (s *StageExecutor) getCreatedBy(node *parser.Node, addedContentSummary stri
 					continue
 				}
 				mountOptionSource = mountInfo.Source
+				mountOptionSource, err = imagebuilder.ProcessWord(mountOptionSource, s.stage.Builder.Arguments())
```

Comment:
Is line 1995 still necessary?

Reply:
Yes since this only processes `source` before passing it to `generatePathChecksum` at other places it is still needed.

Reply:
Ooops, I meant linet 1898

mountOptionSource = mountInfo.Source

I need updated glasses, and it looks like you figured that out.  SGTM.